File: ex-2.2/ex-2.2 (Python)/ex-2.2.py
v1 = int(input ('Digite um numero: '))
v2 = int(input ('Digite outro: '))

print ('Esses são os valores compreendidos entre {} e {}:\n'.format(v1, v2))
if v1 > v2:
    a = v2 + 1
    while a < v1:
        print (a)
        a += 1
if v2 > v1:
    a = v1 + 1
    while a < v2:
        print (a)
        a += 1

File: ex-2.2/ex-2.2 (C)/ex-2.2.c
#include <stdio.h>
#include <stdlib.h>

int main (void){

	int a, v1, v2, maior, menor;
	
	printf ("Digite dois numeros: ");
	scanf ("%d %d", &v1, &v2);
	
	printf ("Esses sao os numeros compreendidos entre %d e %d: \n", v1, v2);

	if (v1 > v2){
		maior = v1;
		menor = v2 + 1;
	} else if (v2 > v1){
		maior = v2;
		menor = v1 + 1;
	}
	while (menor < maior){
		printf ("%d \n", menor);
		++menor;
	}

	return 0;
}
